src/regfox/regfox_api.js

```diff
@@ -123,4 +125,41 @@ const getRegistrationInfoUrl = (id) => {
   return REGFOX_GET_REGISTRATION_URL.replace('${id}', id);
 };
 
-export { getRegistrationInfo, searchRegistrations, exchangeBearerToken, login, REGFOX_GRAPHQL_URL, REGFOX_EXCHANGE_TOKEN_URL, getRegistrationInfoUrl };
+/**
+ * Marks a registrant (technically a transaction of an order of a registered user) as completed (which means fully paid).
+ * See https://gist.github.com/raytingtw/8d0a083dc130c40818d71606e41b884e.
+ * Returns a promise Error if the network is down.
+ *
+ * All of these parameters can be pulled straight from the registrationInfo.
+ *
+ * @param {int} formId is the form (webpage) the user signed up with, could be hardcoded to 372652
+ * @param {int} registrationId of the registrant (**not** the Id), I think sometimes this is called the OrderId
+ * @param {int} transactionId of the registrant
+ * @param {string} id of registration (**not** the RegistrationId)
+ * @param {string} bearerToken the bearer token of the logged in user
+ */
+const markRegistrationComplete = async (formId, registrationId, transactionId, id, bearerToken) => {
```

Comment:
This feels like it's teetering towards a destructured object input instead of a line of IDs. Not sure if 5 is the line to draw, might be okay.

Reply:
Interesting, I'm very much used to [checkstyle default maximum](https://checkstyle.sourceforge.io/apidocs/src-html/com/puppycrawl/tools/checkstyle/checks/sizes/ParameterNumberCheck.html#line.125) of 7 arguments before switching to an object.

I assumed eslint was the same, but it appears they actually set [the default max](https://eslint.org/docs/rules/max-params) to 3! The reason we aren't getting a notification on that is because we are using Google Style which has the [check turned off](https://github.com/google/eslint-config-google/blob/master/index.js#L244)!

Well I'm happy to set the max params to 5, seems like a reasonable middle ground between all three of those options.

Reply:
Just kidding, set it to 6 so that your constructor here doesn't trigger a failure. 

https://github.com/anthroarts/RegExtension/blob/e7b8757a6b700b9a94070aa8e69917821521b4bf/src/regdesk/label_builder.js#L37